openquakeplatform/openquakeplatform/static/js/irv_viewer.js

```diff
@@ -14,1043 +14,850 @@
+    $('#projectDef-spinner').hide();
+    $('#iri-spinner').hide();
+    map = new L.Map('map', {
+        minZoom: 2,
+        scrollWheelZoom: false,
+        attributionControl: false,
+        maxBounds: new L.LatLngBounds(new L.LatLng(-90, -180), new L.LatLng(90, 180)),
+    });
+    map.setView(new L.LatLng(10, -10), 2).addLayer(baseMapUrl);
 
-            iriCopy = jQuery.extend(true, {}, iriIndicator);
-            var aalKey;
-            var aalValue;
-            var sviKey;
-            var sviValue;
-
-            // Compute IRI from PRI and SVI
-            // TODO expand this to use proper IRI function...
-            function getNewValues(k) {
-                aalValue = aalIndicator[k];
-                sviValue = sviIndicator[k];
+    // Slider
+    $(function() {
+        $( '#slider-vertical' ).slider({
+            orientation: 'vertical',
+            range: 'min',
+            min: 0,
+            max: 100,
+            value: 16.666,
+            slide: function( event, ui ) {
+                $( '#econ-weight' ).val( ui.value );
             }
+        });
+        $( '#econ-weight' ).val( $( '#slider-vertical' ).slider( 'value' ) );
+    });
 
-            for (var i = 0; i < municipality.length; i++) {
-                tmp = 0;
-                for(var k in iriIndicator) {
-                    getNewValues(k);
-                    tmp = (sviValue * aalValue);
-                    iriIndicator[k] = tmp;
+    $('#map-tools').append(
+        '<select id="svir-project-list">'+
+            '<option selected disabled>Select Project</option>'+
+        '</select>'
+    );
+
+    $('#map-tools').append(
+        '<select id="thematic-map-selection">'+
+            '<option>Select Indicator</option>'+
+        '</select>'
+    );
+
+    $('#thematic-map-selection').css({ 'margin-bottom' : 0 });
+    $('#svir-project-list').css({ 'margin-bottom' : 0 });
+    $('#region-selection-list').css({ 'margin-bottom' : 0 });
+    $('#svir-project-list').hide();
+    $('#region-selection-list').hide();
+    var SVIRLayerNames = [];
+    var url = "/geoserver/ows?service=WFS&version=1.0.0&REQUEST=GetCapabilities&SRSNAME=EPSG:4326&outputFormat=json&format_options=callback:getJson";
+
+    // Get layers from GeoServer and populate the layer selection menu
+    $.ajax({
+        url: url,
+        contentType: 'application/json',
+        success: function(xml) {
+            //convert XML to JSON
+            var xmlText = new XMLSerializer().serializeToString(xml);
+            var x2js = new X2JS();
+
+            var jsonElement = x2js.xml_str2json(xmlText);
+            var featureType = jsonElement.WFS_Capabilities.FeatureTypeList.FeatureType;
+
+            // Find the SVIR keywords
+            var stringToLookFor = 'SVIR_QGIS_Plugin';
+            for (var i = 0; i < featureType.length; i++) {
+                if (featureType[i].Keywords.indexOf(stringToLookFor) > -1) {
+                    var tempTitle = featureType[i].Title;
```

Comment:
It looks like you are not using tempTitle and tempName, so they could be removed